reconcile/ocm_cluster_admin.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import logging
+from typing import Any, Mapping
+
+from reconcile import queries
+from reconcile.status import ExitCodes
+from reconcile.utils.ocm import OCMMap
+from reconcile.ocm.utils import cluster_disabled_integrations
+
+QONTRACT_INTEGRATION = "ocm-cluster-admin"
+
+
+def _cluster_is_compatible(cluster: Mapping[str, Any]) -> bool:
+    return (
+        cluster.get("ocm") is not None
+        and cluster.get("clusterAdminAutomationToken") is not None
+    )
+
+
+def run(dry_run, thread_pool_size=10):
+    settings = queries.get_app_interface_settings()
+    clusters = queries.get_clusters()
```

Comment:
Is this similar?
https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/gql_definitions/common/clusters_minimal.gql

Reply:
Ah I see it is used in OCMap

Reply:
you tell me: https://github.com/app-sre/qontract-reconcile/blob/7ee5f964600b4c8972e0f84abfbce7d05438a283/reconcile/queries.py#L925-L932

Reply:
I was more about adding another query that is not minimal and rely on fragments for the similarities :)

However, doesnt make sense now because it is used in OCMap ...